torch/autograd/function.py

```diff
@@ -550,7 +551,14 @@ def apply(cls, *args, **kwargs):
 
 
 def once_differentiable(fn):
+    # In order for the function returned by once_differentiable to work with Dynamo,
+    # Dynamo needs to be able to inline through its return to prove safety.
+    # However, Dynamo thinks the function returned by once_differentiable is in a
+    # SKIPFILES, because this file is in SKIPFILES, which prevents inlining. We apply
+    # the following decorator (once per function) to force inlines.
+    @torch._dynamo_utils.force_inline
     @functools.wraps(fn)
+    @torch._dynamo_utils.force_inline
```

Comment:
this line is duplicated?

Reply:
This is intentional, we need to force_inline on `wrapper` and force_inline on the function produced by functools.wraps